src/constants.js

```diff
@@ -0,0 +1 @@
+export const githubOwner = 'code4sac'
```

Comment:
I'm hoping we can accumulate some variables that will change between groups in this file so that we don't need to change copy and variables throughout the whole codebase when adapting this website to other groups/chapters.